### PR TITLE
Various spacing fixes

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
retest this please

File: lib/cfc_m9k_diff.py [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
+LUA_FOOTER = 'end )'
 
 OUTPUT_DIR = 'test'
 
-SENT_FIRST_LINE = '    local sent = scripted_ents.GetStored("{}").t\n'
+SENT_FIRST_LINE = '    local sent = scripted_ents.GetStored( "{}" ).t\n'
 SENT_VAR_LINES  = '    sent.{} = {}\n'
-WEP_FIRST_LINE  = '    local weapon = weapons.GetStored("{}")\n'
+WEP_FIRST_LINE  = '    local weapon = weapons.GetStored( "{}" )\n'
 WEP_VAR_LINES   = '    weapon.{} = {}\n'
```
Comment (justinmuskopf) [inline lib/cfc_m9k_diff.py:19]: nice

**Review by justinmuskopf — APPROVED** [review]
I _might've_ skimmed the last ~90 files

**Comment by sarahsturgeon** [comment]
retest this please

**Pull request merged**